index.html

```diff
@@ -2,9 +2,45 @@
 <html>
   <head>
     <meta charset="utf-8">
-    <title></title>
+    <link rel="stylesheet" href="app.css" media="screen">
+    <title>Octocat Grid</title>
   </head>
   <body>
 
+  <div class="cat-column">
+    <img class="octocats" src="https://octodex.github.com/images/filmtocat.png" alt="film cat"/>
+    <p class="number">#115</p>
+      <div class="created">
+      <p>the <span>Filmcat</span> by
+      <img class="avatar" src="http://lorempixel.com/100/100/" alt="lorem"/></p>
+      <hr>
```

Comment:
`hr` should be used in a very limited way and only to separate between elements in a group. Use `border-bottom` instead if you want to make a border across things. Looks like that border would be at the bottom of the entire column.